### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by parasyte**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4325)

### CODE REVIEW

File: docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -244,7 +244,7 @@ interface between the roles.
 If you do not have such a separation of responsibilities in your organization,
 you may not need to use Pod Disruption Budgets.
 
-## How to perform Distruptive Actions your Cluster
+## How to perform Disruptive Actions your Cluster
```
Comment (chenopis) [inline docs/concepts/workloads/pods/disruptions.md]: This should probably be "How to perform Disruptive Actions `on` your Cluster".

**Comment by parasyte** [comment]
@chenopis Nice catch. I've updated the PR to address the grammatical error. Still awaiting a response from the Linux Foundation helpdesk email in regard to their inability to find a match for our corporate CLA...

**Comment by chenopis** [comment]
@parasyte Ok, no problem. Just ping me once the CLA issue is resolved, and I can merge this. Cheers

**Comment by parasyte** [comment]
@chenopis Thanks for your patience. I was finally able to resolve the issue with the CLA, and it should be good to go now!

**Pull request merged**